libs/initScripts/local.js

```diff
@@ -0,0 +1,124 @@
+const initAudiusLibs = require('../examples/initAudiusLibs')
+const { distributeTokens } = require('./helpers/distributeTokens')
+const { setServiceVersion } = require('./helpers/version')
+const { registerLocalService, queryLocalServices, getStakingParameters } = require('./helpers/spRegistration')
+const { deregisterLocalService } = require('./helpers/spRegistration')
+const { getClaimInfo, fundNewClaim } = require('./helpers/claim')
+
+const serviceTypeList = ['discovery-provider', 'creator-node', 'content-service']
+const spDiscProvType = serviceTypeList[0]
+const spCreatorNodeType = serviceTypeList[1]
+const discProvEndpoint1 = 'http://docker.for.mac.localhost:5000'
+const creatorNodeEndpoint1 = 'http://docker.for.mac.localhost:4000'
+const creatorNodeEndpoint2 = 'http://docker.for.mac.localhost:4010'
+const versionStr = '0.1.0'
+
+const throwArgError = () => {
+  throw new Error(`missing argument - format: node examples/initiateStakingOperations.js [
```

Comment:
nit - should probably be node examples/local

Reply:
done